Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Controls/Krypton.Toolkit.Suite.Extended.Controls.csproj

```diff
@@ -28,7 +28,18 @@
     <!--<PackageReference Include="Krypton.Toolkit" Version="5.550.2103" />-->
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Classes\" />
+    <Compile Update="Components\Scroll Bars\Skins\HScrollSkin.cs">
+      <SubType>Component</SubType>
```

Comment:
Thisis an "Old style" csproj layout.. Please chnage to the new to allow TFM's etc.

Reply:
Sorted in subsequent